kolibri/core/assets/src/views/UserTypeDisplay.vue

```diff
@@ -54,11 +32,13 @@
       },
     },
     $trs: {
+      /* eslint-disable kolibri/vue-no-unused-translations */
```

Comment:
It may be helpful to briefly comment that the string needs to stay here because we use `UserTypeDisplay` from the `crossComponentTranslator`

Reply:
Sure, let me do that